### ISSUE TITLE
pandas.Grouper documentation thin

### ISSUE THREAD

**Issue by eXcuvator** [pydata/pandas#9789]
The documentation of http://pandas.pydata.org/pandas-docs/dev/generated/pandas.Grouper.html is quite thin, when it comes to *additional kwargs to control time-like groupers (when freq is passed)*:

- convention is not described at all
- default values are not described for any of the additional kwargs

**Comment by jreback** [comment]
*time-like grouping* is a *resample*, so the same frequency operators are accepted, including all ``DateOffsets``, see [here](http://pandas.pydata.org/pandas-docs/dev/timeseries.html#dateoffset-objects), so certainly would accept a link to this; don't want to repeat things here however.

**Comment by Winterflower** [comment]
I would like to pick this up, but the link OP posted http://pandas.pydata.org/pandas-docs/dev/generated/pandas.Grouper.html  redirects to the pandas frontpage so I am not entirely sure, which page he/she would like to see documented. Perhaps this http://pandas-docs.github.io/pandas-docs-travis/groupby.html ?

**Comment by jorisvandenbossche** [comment]
Ah, yes, the API docs are not included in the latest dev docs, you can look just at the stable docs:  http://pandas.pydata.org/pandas-docs/stable/generated/pandas.Grouper.html

**Comment by Winterflower** [comment]
Thank you @jorisvandenbossche . I'll try to expand the section based on the feedback from @jreback and OP. 
The docstring could certainly do with a bit of tidying. 
At first I thought that the 'syntactic sugar' part here 
```python
df.groupby(Grouper(key='A')) : syntactic sugar for df.groupby('A')
```
was part of the code (the syntax highlighting on the 'for' is misleading).

**Comment by jorisvandenbossche** [comment]
Yes, in those cases everything after the `:` which is not code but explanation should be on separate lines to make this more clear

**Comment by jreback** [comment]
closed by #9789
